netlify.toml

```diff
@@ -5,7 +5,7 @@
     command = "yarn build --environment=production"
 
 [context.deploy-preview]
-    command = "yarn build --environment=development"
+    command = "yarn build --environment=review"
```

Comment:
this is for the ember app, which does not have a `review` environment defined in the config

Reply:
nvm